libcorn/Cargo.toml

```diff
@@ -24,6 +25,7 @@ wasm-bindgen = { version = "0.2.83", optional = true }
 serde-wasm-bindgen = { version = "0.5.0", optional = true }
 console_error_panic_hook = { version = "0.1.7", optional = true }
 wee_alloc = { version = "0.4.5", optional = true }
+mlua = { version = "*", features = ["luajit", "vendored", "module", "macros"], optional = true }
```

Comment:
Please use an exact version rather than a wildcard, as otherwise this could lead to broken builds. Cargo should be smart enough to pick newer versions based on semantic versioning where possible.